tests/PHPUnit/Unit/Validator/MollieGatewayConfigValidatorTest.php

```diff
@@ -0,0 +1,286 @@
+<?php
+
+/*
+ * This file is part of the Sylius Mollie Plugin package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\MolliePlugin\PHPUnit\Unit\Validator;
+
+use Doctrine\ORM\PersistentCollection;
+use Sylius\MolliePlugin\Entity\MollieGatewayConfigInterface;
+use Sylius\MolliePlugin\Entity\MollieMinMaxInterface;
+use Sylius\MolliePlugin\Validator\Constraints\MollieGatewayConfigValidator;
+use Sylius\MolliePlugin\Validator\Constraints\MollieGatewayConfigValidatorType;
+use Symfony\Component\Validator\ConstraintValidatorInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+final class MollieGatewayConfigValidatorTest extends ConstraintValidatorTestCase
+{
+    private MollieGatewayConfigInterface $mollieGatewayConfig;
+
+    private MollieMinMaxInterface $configAmountLimits;
+
+    protected function setUp(): void
```

Comment:
This is the only protected method that should be on top, everything else should go after public ones